recipe/meta.yaml

```diff
@@ -11,32 +11,29 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  noarch: python
+  skip: true  # [py<37]
   number: 0
-  skip: True  # [py<36]
   script: {{ PYTHON }} -m pip install . -vv
 
 requirements:
   host:
     - python
     - pip
-    - setuptools
     - wheel
   run:
-    - python >=3.6
+    - python
     - aniso8601 >=0.82
     - jsonschema
     - flask >=0.8,!=2.0.0
     - werkzeug !=2.0.0
     - pytz
-    - six >=1.3.0
 
 test:
   imports:
     - flask_restx
   requires:
     - pip
-    - python <3.10
+    - python
```

Comment:
`python` can be removed from `test/requires. `It's already in `run`. All run dependencies will be in the test environment.